docker/nginx/default.conf

```diff
@@ -1,13 +1,26 @@
 server {
-  listen 80 default_server;
+  listen 80 default_server reuseport;
 
   keepalive_timeout 70;
 
   index index.php;
 
-  location / {
-    root /usr/share/nginx/html;
-    try_files $uri /index.php$is_args$args;
+  root /usr/share/nginx/html;
+  try_files $uri /index.php$is_args$args;
+
+  # security headers
+  add_header X-XSS-Protection          "1; mode=block" always;
+  add_header X-Content-Type-Options    "nosniff" always;
```

Comment:
Small comment retrieved from MDN:

> Starting with Firefox 72, top-level documents also avoid MIME sniffing (if Content-type is provided). This can cause HTML web pages to be downloaded instead of being rendered when they are served with a MIME type other than text/html. Make sure to set both headers correctly.

No action needed IMO.

Reply:
Do you mean, remove it?

Reply:
No, just a heads up if we are bit by the problem they described (web pages being downloaded instead of rendered).

No action needed here.